### ISSUE TITLE
Clone Repository doesn't do anything

### ISSUE THREAD

**Issue by it-is-just-me** [microsoft/vscode-remote-release#4286]
Issue Type: <b>Bug</b>

1. Start VS Code Insider
2. Run "Remote-Containers: Clone Repository in Container Volume"
-> nothing happens, no dialog appears

Expected: open next dialog with input for GitHub url (as described in step 2 here https://code.visualstudio.com/docs/remote/containers#_quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume )

Extension version: 0.154.2
VS Code version: Code - Insiders 1.53.0-insider (3ceb3a100ed0ad5f3238b5022d9d64353a1cc31e, 2021-01-11T05:14:18.990Z)
OS version: Darwin arm64 20.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|undefined|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 3|
|Memory (System)|16.00GB (0.64GB free)|
|Process Argv|--crash-reporter-id 3dee17f6-00be-4652-9115-1ae006792edf|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
openlogontheside:30221882
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30208929
wsl2promptcf:30219163
vstry244:30230484
pythonvsdeb440:30224570
unusedprompt:30219164
folderexplorer:30219166
openfilemenu:30219168
pythonvsded773:30223139
core-portspanel:30233467

```

</details>

**Comment by chrisflatley** [comment]
I have the same but on the standard release branch:

Extension version 0.154.2 (as above)

```
Version: 1.52.1
Commit: ea3859d4ba2f3e577a159bc91e3074c5d85c0523
Date: 2020-12-16T16:30:02.420Z (3 wks ago)
Electron: 9.3.5
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 20.2.0
```

**Comment by chrmarti** [comment]
Any errors in the console (`Help` > `Toggle Developer Tools`)?

**Comment by chrisflatley** [comment]
I opened the Dev Tools and the Console had an warning `git not installed` (it was already there before I tried cloning into the container volume).

When I checked on the command-line, git didn't actual work - it must have been corrupted by an Apple upgrade in the last week??

I re-installed with `xcode-select --install`. 

Now VSCode 'Clone repository into container volume` works again as before.

Perhaps, VS Code `clone into container volume` needs its own check and warning if git is not available?
